docs/pfsense_installerconfig.md

```diff
@@ -34,7 +110,7 @@ applied to this file to eliminate interactive dialog options.
     #TERM=${TERM:-vt100}
     ```
 
-3. Add reboot command after installerconfig finishes:
+1. Add reboot command after installerconfig finishes:
```

Comment:
This is no longer needed AFAIK, reboot is already present there in upstream

Reply:
Removed